packages/core/content-releases/server/src/services/release.ts

```diff
@@ -76,6 +90,10 @@ const createReleaseService = ({ strapi }: { strapi: LoadedStrapi }) => ({
       populate: { release: { fields: ['id'] }, entry: { fields: ['id'] } },
     });
   },
+  deleteAction(actionId: DeleteReleaseAction.Request['params']['actionId']) {
+    // An action can only belong to a single release
+    return strapi.entityService.delete(RELEASE_ACTION_MODEL_UID, actionId);
```

Comment:
ℹ️ Should we still do where releaseId even though an action can only belong to one release?

Reply:
IMO I would add it, because in this way we make sure the url is always right.

For example, you have:
* Release 1, with actions 1, 2 3
* Release 2, with actions 4, 5, 6

If you don't add the where, you can do something like this: `DELETE /content-releases/1/actions/6` and you would delete the action from another release different to the one on your URL

It's true that there is not a real difference in terms of security, but I think is good to keep consistency

Reply:
oh yeah 🤦‍♂️ that was stupid, my bad